server/database/schema.sql

```diff
@@ -31,4 +31,12 @@ create table stations (
 create table socket (
   id int unsigned primary key auto_increment not null,
   type varchar(155) not null
+);
+
+create table usermessage (
+  firstname varchar(80) not null,
+  lastname varchar(80) not null,
+  email varchar(255) not null unique,
```

Comment:
Adding the "unique" constraint to the email in this case will cause issues as a person will only be able to ever send one message with the same email address.